src/App/Documentation/components/Toast/__snapshots__/index.test.js.snap

```diff
@@ -270,6 +270,16 @@ exports[`Components: Toast Overview calls toast on click 1`] = `
       Click for toast!
     </button>
   </ComponentPreview>
+  <Alert
+    type="warning"
+  >
+    <h5>
+      Content restriction
+    </h5>
+    <p>
+      Don’t put too much text in toasts as they are meant to be visible for only a short amount of time. Due to the time restriction you should also avoid putting vital information in them.
```

Comment:
What should be used instead of a toast if one has vital or a lot of information to present?